src/Microsoft.AspNet.Mvc.TagHelpers/JavaScriptHelpers.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Text;
+
+namespace Microsoft.AspNet.Mvc.TagHelpers
```

Comment:
There's generally a high bar for having internal types in Mvc. Put it in an internal namespace `Microsoft.AspNet.Mvc.TagHelpers.Internal` unless there're specific reasons not to.